src/views/CreateShareStructure.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+   <section class="mt-10">
+      <header id="share-structure-header">
+        <h2>1. Create Your Authorized Share Structure</h2>
+      </header>
+    </section>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component({})
+export default class CreateShareStructure extends Vue {}
```

Comment:
one thing i've noticed from trying to resume drafts is we need a way to re-validate after reloading a draft. I'm not sure the best solution on this, but i've fiddle around with the validating methods in a `mounted` hook in the frameworks. Just something to think about, maybe you have a better idea :)